README.md

```diff
@@ -31,14 +69,393 @@ Permissions to allow the user to interact with the support center:
+    name: example-accountpool
+    uid: 9979786a-a8cb-11e9-a2a3-2a2ae2dbcce4
+  resourceVersion: "49984188"
+  selfLink: /apis/aws.managed.openshift.io/v1alpha1/namespaces/aws-account-operator/accounts/osd-{accountName}
+  uid: a07a6867-9dac-11e9-b4bb-0e6ed767b7c0
+spec:
+  awsAccountID: "0000000000"
+  claimLink: example-link
+  iamUserSecret: osd-{accountName}-secret
+status:
+  claimed: true
+  conditions:
+  - lastProbeTime: 2019-07-03T16:14:50Z
+    lastTransitioNTime: 2019-07-03T16:14:50Z
+  	message: Attempting to create account
+ 	  reason: Creating
+ 	  status: "True"
+ 	  type: Creating
+  - lastProbeTime: 2019-07-03T16:18:55Z
+    lastTransitioNTime: 2019-07-03T16:18:55Z
+    message: Account pending AWS limits verification
+    reason: PendingVerification
+    status: "True"
+    type: PendingVerification
+  - lastProbeTime: 2019-07-05T13:19:32Z
+    lastTransitioNTime: 2019-07-05T13:19:32Z
+    message: Account ready to be claimed
+    reason: Ready
+    status: "True"
+    type: Ready
+  state: Ready
+  supportCaseID: case-000000000-muen-2019-000000000000
+```
+
+
+## 2.3. AccountClaim CR
+
+The AccountClaim CR holds the required data for cluster provisioning to build the cluster 
+
+```yaml
+apiVersion: aws.managed.openshift.io/v1alpha1
+kind: AccountClaim
+metadata:
+  creationTimestamp: 2019-07-16T13:52:02Z
+  generation: 2
+  labels:
+    api.openshift.com/id: 00000000000000000000
+    api.openshift.com/name: example-link
+  name: example-link
+  namespace: {NameSpace cluster is being built in}
+  resourceVersion: "54324077"
+  selfLink: /apis/aws.managed.openshift.io/v1alpha1/namespaces/uhc-staging-16toh05i9h8ook5d7ej4al7ejec17rug/accountclaims/razevedo-test2
+  uid: b7113d4a-a8cb-11e9-a2a3-2a2ae2dbcce4
+spec:
+  accountLink: osd-{accountName} (From AccountClaim)
+  aws:
+    regions:
+    - name: us-east-1
+  awsCredentialSecret:
+    name: aws
+    namespace: {NameSpace cluster is being built in}
+  legalEntity:
+    id: 00000000000000
+    name: {Legal Entity Name}
+status:
+  conditions:
+  - lastProbeTime: 2019-07-16T13:52:02Z
+    lastTransitionTime: 2019-07-16T13:52:02Z
+    message: Attempting to claim account
+    reason: AccountClaimed
+    status: "True"
+    type: Unclaimed
+  - lastProbeTime: 2019-07-16T13:52:03Z
+    lastTransitionTime: 2019-07-16T13:52:03Z
+    message: Account claimed by osd-creds-mgmt-fhq2d2
+    reason: AccountClaimed
+    status: "True"
+    type: Claimed
+  state: Ready
+```
+# 3. The controllers
+
+## 3.1. AccountPool Controller
+
+The accountpool-controller is triggered by an accountpool CR or an account CR. It is responsible for generating new account CRs. 
+
+It looks at the accountpool CR *spec.poolSize* and it ensures that the number of unclaimed accounts matchs the number of the poolsize. If the number of unclaimed accounts is less then the poolsize it creates a new account CR for the account-controller to process.
+
+### 3.1.1. Constants and Globals
+```
+emailID = "osd-creds-mgmt"
```

Comment:
This will also be used for account name. A small summary of what each const or global is used for may be helpful

Reply:
I agree , but I think we should put a global section in the next refactor

Reply:
This documentation is about the current state not the next refactor. We should have a description of each constant.